rusk/Cargo.toml

```diff
@@ -38,6 +38,7 @@ parking_lot = "0.12"
 rkyv = { version = "0.7", default-features = false, features = ["size_32"] }
 bytecheck = { version = "0.6", default-features = false }
 dirs = "4"
+blake3 = "1"
```

Comment:
Aren't we mostly using `blake2`? Not per se against using `blake3`, but it might be nice to keep things consistent.

Reply:
Piecrust is using blake3 and I had to use the same here so that contract id is compatible (i.e., the same) as in Piecrust. So this is consistent with Piecrust. Actually, I had blake2 before and I has to change to blake3 :)